src/components/SittingImg.jsx

```diff
@@ -1,11 +1,27 @@
 import Image from "next/image";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
 
 import Sitting from "~/Sitting.png";
 
 const SittingImg = () => {
+    const locale = useRouter().locale;
+
+    useEffect(() => {
+        if (locale === "en") {
+            document
+                .querySelector("#sitting-img")
+                .classList.add("scale-x-[-1]");
+        } else {
+            document
+                .querySelector("#sitting-img")
+                .classList.remove("scale-x-[-1]");
+        }
+    }, [locale]);
```

Comment:
First of all, great work!

Here you don't the `router` at all, There is already a handler that handle the document direction in the navbar and it will switch items to the right automatically so this is not needed

Reply:
Hey Abdullah Thanks you so much,
I've added the **router** becasue the img get wrong direction when moving to English language like in the image below:
![flipped](https://user-images.githubusercontent.com/119085953/230430930-4d3d6af3-2541-451a-80bd-c5d173a4b079.png)
I hope you get my point and thanks again for you help